src/durableClient/DurableClient.ts

```diff
@@ -637,6 +640,19 @@ export class DurableClient implements types.DurableClient {
         }
     }
 
+    private getDistributedTracingHeaders(): Record<string, string> {
+        // Get the current active span
+        const currentSpan = OpenTelemetryApi.trace.getSpan(OpenTelemetryApi.context.active());
+
+        // Create the headers object and inject the current span context if it exists
+        const headers: Record<string, string> = {};
+        if (currentSpan) {
+            OpenTelemetryApi.propagation.inject(OpenTelemetryApi.context.active(), headers);
```

Comment:
Should we be reusing the local variable we already created here?
```suggestion
            OpenTelemetryApi.propagation.inject(currentSpan, headers);
```

Reply:
`OpenTelemetryApi.propagation.inject` takes the current active context as the first argument and `currentSpan` is the current active span so I don't think we can use the local variable. I can save `OpenTelemetryApi.context.active()` in another local variable if you prefer that.

Reply:
Oh, I think I may have misread the code previously. You can ignore this suggestion.